src/games/behavmixed.imp

```diff
@@ -243,7 +196,7 @@ void MixedBehaviorProfile<T>::UndefinedToCentroid()
       }
       if (total == (T) 0) {
 	for (int act = 1; act <= m_support.NumActions(pl, iset); act++) {
-	  (*this)(pl, iset, act) = (T) 1.0 / (T) m_support.NumActions(pl, iset);
+	  (*this)(pl, iset, act) = T(1.0) / T(m_support.NumActions(pl, iset));
```

Comment:
Should use `T(1)` instead of `T(1.0)`.  I think `Rational`'s constructor handles this OK but even better to use an int!